doc/test_containers_starting_guide.md

```diff
@@ -0,0 +1,88 @@
+This is a starting reference for adding test containers to the project (probably needed when we add DAPS to integration tests).
```

Comment:
Could add link to create custom Testcontainer
https://java.testcontainers.org/features/creating_container/

Reply:
Disregard this comment, I did not go all the way to the end of document

Reply:
👍